app/assets/stylesheets/administrate/components/_pagination.scss

```diff
@@ -15,3 +15,15 @@
     font-weight: $bold-font-weight;
   }
 }
+
+.pagination-with-total {
+  margin-bottom: $base-spacing * 4;
+
+  .pagination {
+    margin-bottom: $small-spacing;
+  }
+
+  .pagination-total {
+    text-align: center;
```

Comment:
@tysongach `.pagination` isn't the parent of `pagination__total`. This style is needed.